source/mvgApi.js

```diff
@@ -40,8 +42,57 @@ function getDepartures(station, options, apiUrl = defaultApi) {
             }
         });
     });
+}
+
+/**
+ *
+ * @param suggestion
+ * @param options
+ * @param apiUrl
+ * @returns {Promise}
+ * @TODO: add reject and error handling
```

Comment:
:+1:

Reply:
My IDE (PhpStorm) suggests another way of documentation, different from the old one you added. Which way should we stick to?